var/spack/repos/builtin/packages/pumi/package.py

```diff
@@ -41,7 +41,7 @@ class Pumi(CMakePackage):
     variant('shared', default=False, description='Build shared libraries')
     variant('zoltan', default=False, description='Enable Zoltan Features')
     variant('fortran', default=False, description='Enable FORTRAN interface')
-    variant('testing', default=False, description='Enable tests')
+    variant('testing', default=False, description='Enable all tests')
```

Comment:
Not a fan of having variants that enable tests, since tests are not part of the provenance and variants change the hash. In any case, this wwas already in the package, so we can remove this variant in another PR.